CMakeLists.txt

```diff
@@ -78,33 +78,43 @@ ecbuild_bundle( PROJECT fv3-jedi    GIT "https://github.com/jcsda/fv3-jedi.git"
 find_branch_name(REPO_DIR_NAME ioda)
 # When LOCAL_PATH_JEDI_TESTFILES is set to the directory of IODA test files stored
 # in a local directory, ioda-data repo will not be cloned
-if( NOT DEFINED ENV{LOCAL_PATH_JEDI_TESTFILES} AND NOT DEFINED ${GIT_TAG_FUNC} )
```

Comment:
I noticed `${GIT_TAG_FUNC}` is replaced by `GIT_TAG_FUNC`, is the behavior unchanged? (I'm not very familiar with cmake)

Reply:
It's weird cmake behavior. We need to use `DEFINED` without `${}` to check if the variable is defined or not.